db/setup/deployment-mass-import/deployment-mass-import.py

```diff
@@ -0,0 +1,537 @@
+    - output (str): The path to the folder.
+
+    Returns:
+    None
+    """
+    data = validator.PPicture(**pic)
+    extension = name.split(".")[-1]
+    filename = name.removesuffix("." + extension)
+    filePath = f"{output}/{filename}.json"
+    with open(filePath, "w") as json_file:
+        json.dump(pic, json_file, indent=2)
+
+
+def uploadSessionDB(path: str, userID: str):
+    """
+    Upload the session.json file located at the specified path to the database.
+
+    Parameters:
+    - path (str): The path to the session file.
+    - userID (str): The ID of the user.
+
+    Returns:
+    - The ID of the session.
+    """
+    try:
+        # Connect to your PostgreSQL database
+        conn = psycopg.connect(os.getenv("NACHET_DB_URL"))
+
+        # Create a cursor object
+        cur = conn.cursor()
+
+        queries.createSearchPath(conn, cur)
+
+        # Open the file
+        with open(path, "r") as file:
+            data = file.read()
+
+        # Build sessionID
+        sessionID = uuid.uuid4()
+
+        # Execute the INSERT statement
+        # print(userID)
+        query = "INSERT INTO sessions (id,session, ownerID) VALUES (%s,%s, %s)"
+        params = (
+            sessionID,
+            data,
+            userID,
+        )
+        cur.execute(query, params)
+        conn.commit()
+    except:
+        print("Error: could not upload the session to the database")
+        sessionID = None
+    # Retrieve the session id
+    # cur.execute("SELECT id FROM sessions ORDER BY id DESC LIMIT 1")
+    finally:
+        cur.close()
+        conn.close()
+    return sessionID
+
+
+def uploadPictureDB(path: str, sessionID: str, seedID: str):
+    """
+    Uploads the picture file located at the specified path to the database.
+
+    Parameters:
+    - path (str): The path to the picture file.
+    - userID (str): The ID of the user.
+    - sessionID (str): The ID of the session.
+
+    Returns:
+    None
+    """
+    try:
+        # Connect to your PostgreSQL howatabase
+        conn = psycopg.connect(os.getenv("NACHET_DB_URL"))
+
+        # Create a cursor object
+        cur = conn.cursor()
+
+        queries.createSearchPath(conn, cur)
+        # Open the file
+        with open(path, "r") as file:
+            data = file.read()
+
+        # Generate pictureID
+        pictureID = uuid.uuid4()
+
+        # Execute the INSERT statement
+        params = (
+            pictureID,
+            data,
+            sessionID,
+        )
+        query = "INSERT INTO pictures (id,picture, sessionID) VALUES (%s,%s, %s)"
+        cur.execute(query, params)
+        conn.commit()
+
+        id = uuid.uuid4()
+
+        # Create a Picture - Seed relationship
+        param = (
+            id,
+            seedID,
+            pictureID,
+        )
+        query = "INSERT INTO seedpicture (id,seedID,pictureID) VALUES (%s,%s,%s)"
+        cur.execute(query, param)
+        conn.commit()
+    except:
```

Comment:
👍🏻 Instead of just using `except`, you can try to catch specific exceptions that would be raised if your code encounters errors. This is the same as `except Exception` which could hide critical errors that you might want to catch otherwise.